dev-lang/julia/julia-1.7.2.ebuild

```diff
@@ -0,0 +1,199 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit llvm pax-utils toolchain-funcs
+
+# correct versions for stdlibs are in deps/checksums
+# for everything else, run with network-sandbox and wait for the crash
+
+MY_LLVM_V="13.0.0"
+
+DESCRIPTION="High-performance programming language for technical computing"
+HOMEPAGE="https://julialang.org/"
+
+SRC_URI="
+	https://github.com/JuliaLang/julia/releases/download/v${PV}/${P}-full.tar.gz
+	https://github.com/JuliaLang/julia/commit/677ce6d3.patch -> ${PN}-1.7.2-llvm_13_compat_part_1.patch
+	https://github.com/JuliaLang/julia/commit/47f9139e.patch -> ${PN}-1.7.2-llvm_13_compat_part_2.patch
+	https://github.com/JuliaLang/julia/commit/1eb063f1.patch -> ${PN}-1.7.2-llvm_13_compat_part_3.patch
+	https://github.com/JuliaLang/julia/commit/99d4e655.patch -> ${PN}-1.7.2-llvm_13_compat_part_4.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/f8c918b0.patch -> ${PN}-1.7.2-llvm_13_compat_part_5.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/63303980.patch -> ${PN}-1.7.2-llvm_13_compat_part_6.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-libgit-1.2.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-libgit-1.4.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-system-cblas.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-hardcoded-libs.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/make-install-no-build.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-llvm-patchelf.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-libunwind-1.6.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-curl-7.81.patch
+	https://raw.githubusercontent.com/archlinux/svntogit-community/packages/julia/trunk/julia-libblastrampoline-4.patch
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+system-llvm"
+
+RDEPEND="
+	system-llvm? ( sys-devel/llvm:13=[llvm_targets_NVPTX(-)] )
+"
+LLVM_MAX_SLOT=13
+
+RDEPEND+="
+	app-arch/p7zip
+	dev-libs/gmp:0=
+	dev-libs/libgit2:0
+	>=dev-libs/libpcre2-10.23:0=[jit,unicode]
+	dev-libs/mpfr:0=
+	>=dev-libs/libutf8proc-2.6.1:0=[-cjk]
+	>=dev-util/patchelf-0.13
+	>=net-libs/mbedtls-2.2
+	<net-misc/curl-7.81.0[http2,ssh]
```

Comment:
Maybe we can drop this constraint since we have 
> https://github.com/gentoo/gentoo/pull/25496/files#diff-426382c6ef8f658f80a5a7c5387ab4d45770043cff9c44104b77704289a7a06bR31

Reply:
yes, but do not forget USE :D `[http2,ssh]`